### PR TITLE
Add solutions page

### PR DESCRIPTION
**Pull request by Smoke-ck**

### CODE REVIEW

File: solutions.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                <img src="assets/images/solutions/{{ item.image }}.webp" alt="Approach">
+              </div>
+              <div class="timeline-item__content">
+                <div class="timeline-item__content-title inria">{{ item.title}}</div>
+                <div class="timeline-item__content-description">{{ item.description}}</div>
+              </div>
+              <div class="timeline__processes">
+                {% for process in item.processes%}
+                  <div class="timeline__processes-item">
+                    {{process.item}}
```
Comment (katatsu12) [inline solutions.html:135]: please check the images:

image

image

File: solutions.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    <div class="arrow">→</div>
+                  </div>
+                {% endfor %}
+              </div>
+            </div>
+          {% endfor %}
+        </div>
+      </div>
+      <a class="cto-button" href="/services.html">{{ site.data.solutions.staff_augmentation.more_button }}</a>
```
Comment (katatsu12) [inline solutions.html:144]: footer is moving when hover on the button
image

File: solutions.html (hunk 1 above)
Comment (Smoke-ck) [reply solutions.html:135]: Will check

File: solutions.html (hunk 2 above)
Comment (Smoke-ck) [reply solutions.html:144]: Ok

**Comment by katatsu12** [comment]
looks like some text here is missing:
image

text on buttons black but white on the design:
image

**Review by katatsu12 — APPROVED** [review]

**Pull request merged**